src/components/Navigation.module.css

```diff
@@ -28,6 +29,8 @@
   margin: 0;
   flex-shrink: 0;
   display: none;
+  box-shadow: 0 0 20px #555;
```

Comment:
```suggestion
  box-shadow: 0 0 20px #777;
```
After playing with it, this seems to look better.

Reply:
Definitely looks better.